nixos/modules/services/desktops/flatpak.md

```diff
@@ -25,7 +23,36 @@ in other cases, you will need to add something like the following to your
 }
 ```
 
-Then, you will need to add a repository, for example,
+## NixOS-integrated Flatpak Package Management {#nixos-integrated-flatpak-package-management}
+NixOS now supports integrated management of Flatpak packages. You can specify
+Flatpak packages to be installed or removed through your {file}`configuration.nix`:
+```nix
+{
+  services.flatpak = {
+    enable = true;
+    packages = [ "org.blender.Blender" "net.ankiweb.Anki" ];
+    removeUnmanagedPackages = true;
+    automaticUpdates = false;
+  };
+}
+```
+
+This configuration will:
+- Enable Flatpak
+- Ensure the specified packages are installed (Blender and Anki in this example)
+- Remove any Flatpak packages not listed in the `packages` option (if `removeUnmanagedPackages` is true)
+- Control whether automatic updates are performed for Flatpak packages
+
+Important notes:
+- Package versions are managed by Flatpak's servers, not nixpkgs.
+- The `removeUnmanagedPackages` option defaults to `false`.
+- The `automaticUpdates` option defaults to `false`. When set to `true`, it allows automatic updates of installed Flatpak packages.
+
+Be aware that updates to Flatpak packages may occur independently of your NixOS system updates. If you encounter issues with a Flatpak package, it's recommended to check with the upstream project first before reporting issues to NixOS.
```

Comment:
That advise is generally not true. If the issue is related to the programs code then people should go upstream but it could also be that someone is missing a portal or something

Reply:
Thanks for the reviews, I'll try to make modifications accordingly